### PR TITLE
fix: fix command of the cocoapods

### LINKED ISSUE #93: Error: /bin/sh: pod: command not found
As title.

```sh
ℹ️  STARTED: "wipe system iOS Pods cache"
error

Task "wipe system iOS Pods cache"
Error: /bin/sh: pod: command not found
.
Exit code: 127

.
Error:

Task "wipe system iOS Pods cache"
Error: /bin/sh: pod: command not found
.
Exit code: 127
```

### PR DESCRIPTION
**Pull request by whck6**
Signed-off-by: Wenhao Ho <wh.ho@outlook.com>

Fixed #93

### CODE REVIEW

**Comment by patik** [comment]
Shouldn't it use `&&` ?

File: source/internals/tasks.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const tasks = {
   },
   wipeSystemiOSPodsCache: {
     name: 'wipe system iOS Pods cache',
-    command: 'cd ios & pod',
+    command: 'cd ios & bundle exec pod',
```
Comment (patik) [inline source/internals/tasks.js:15]: Shouldn't it be like this?

```suggestion
    command: 'cd ios && bundle exec pod',
```

Comment (whhotw) [reply source/internals/tasks.js:15]: Yes, you're right.

**Comment by whhotw** [comment]
@pmadruga Hi, Could you help to merge this? I was thinking someone might be able to install a `pod` with the specific path like: `vender/**` and even add this change doesn't have side-effected with installed it with global of the users.

**Comment by azedo** [comment]
Hi, are there any updates on this? I was hoping to add the lib to my project, but I need this to be fixed otherwise I get version conflicts.

**Comment by whck6** [comment]
@azedo I thought that would work even if the change doesn't apply.

**Pull request merged**